src-tauri/src/chess/types.rs

```diff
@@ -171,48 +168,18 @@ impl Display for Piece {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         let code = match *self {
             Piece::None => "__",
-            Piece::Pawn(color, ..) => {
-                if color == Color::White {
-                    "wP"
-                } else {
-                    "bP"
-                }
-            }
-            Piece::King(color, ..) => {
-                if color == Color::White {
-                    "wK"
-                } else {
-                    "bK"
-                }
-            }
-            Piece::Queen(color, ..) => {
-                if color == Color::White {
-                    "wQ"
-                } else {
-                    "bQ"
-                }
-            }
-            Piece::Bishop(color, ..) => {
-                if color == Color::White {
-                    "wB"
-                } else {
-                    "bB"
-                }
-            }
-            Piece::Knight(color, ..) => {
-                if color == Color::White {
-                    "wN"
-                } else {
-                    "bN"
-                }
-            }
-            Piece::Rook(color, ..) => {
-                if color == Color::White {
-                    "wR"
-                } else {
-                    "bR"
-                }
-            }
+            Piece::Pawn(Color::White, ..) => "wP",
```

Comment:
What are the two dots doing here?

Reply:
That's short hand for "ignore any other variables I might pull out of this pattern"

So is the equivalent of 

```rust

Piece::Pawn(Colour::White, _, _) => "wP",

```